config/routes.rb

```diff
@@ -3,8 +3,16 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  root to: redirect('/users', status: 302)
-  resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+  # root to: redirect('/users', status: 302)
+  # resources :users, only: %i[index show] do
+  #   resources :posts, only: %i[index show]
+  # end
```

Comment:
- [ ] Please avoid leaving old pieces of commented code as they make your codebase look ugly and less readable! 🤓 🔧

Reply:
Thank you for your nice feedback @anagudelogu ❤️